modules/claims_api/lib/bd/bd.rb

```diff
@@ -58,13 +58,14 @@ def upload(claim:, pdf_path:, doc_type: 'L122', file_number: nil)
     # @return {parameters, file}
     def generate_upload_body(claim:, doc_type:, pdf_path:, file_number: nil)
       payload = {}
+      file_name = "526EZ_#{claim.auth_headers['va_eauth_firstName']}_#{claim.auth_headers['va_eauth_lastName']}"
       data = {
```

Comment:
@stiehlrod  Is the name suppose to include the claim_id as well, or did that change?